test/benchmarks/driver_bench/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "driver_bench",
+  "version": "1.0.0",
+  "scripts": {
+    "prestart": "tsc",
+    "start": "node lib/main.mjs"
+  },
+  "devDependencies": {
+    "@types/node": "^22.13.0"
```

Comment:
I'm not entirely opposed to a subpackage for this, but we've encountered friction in the past (with our eslint plugin in BSON) with keeping this dependency up-to-date in our quarterly dependency upgrades.

Is this approach significantly simpler than just compiling the benchmarks and then running them from the run-benchmark script?

Reply:
I just wanted to capture the "lifecycle" scripts into one command so stopped forgetting to recompile locally while developing. 

I don't actually want a subpackage here, we don't need to add the node types as a dep, removed!